packages/cli/test/unit/commands/domains/transfer-in.test.ts

```diff
@@ -16,6 +16,24 @@ describe('domains transfer-in', () => {
     });
   });
 
+  describe('--help', () => {
+    it('tracks telemetry', async () => {
+      const command = 'domains';
+      const subcommand = 'transfer-in';
+
+      client.setArgv(command, subcommand, '--help');
+      const exitCodePromise = domains(client);
+      await expect(exitCodePromise).resolves.toEqual(2);
+
+      expect(client.telemetryEventStore).toHaveTelemetryEvents([
+        {
+          key: 'flag:help',
+          value: `${command}:transferIn`,
```

Comment:
I could transform this to precisely match the terminal value passed but that doesn't seem worth the additional code.

Reply:
Please change this to precisely match, so it matches our existing subcommand event.